addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -391,6 +391,167 @@ options.registry.parallax = options.Class.extend({
     },
 });
 
+options.registry.facebook_page = options.Class.extend({
+    template: 'website.facebook_page_dialog',
+    xmlDependencies: ['/website/static/src/xml/website.facebook_page.xml'],
+
+    /**
+     * @override
+     */
+    start: function () {
+        var self = this;
+
+        this.$target.on('click', '.o_add_facebook_page', function (event) {
+            event.preventDefault();
+            self.fbPageOptions();
+        });
+        return $.when(this._setParams(), this._super.apply(this, arguments));
+    },
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Open dialog to configure facebook page options.
+     */
+    fbPageOptions: function () {
+        var self = this;
+
+        var $dialog = new Dialog(null, {
+            title: _t('Facebook page'),
```

Comment:
As the string `Facebook page` will be shown to the user, it is considered a non-technical string and our convention is to use `"` instead of `'` for them.